### ISSUE TITLE
Current Page breadcrumb fails Use of Color

### ISSUE THREAD

**Issue by mbgower** [carbon-design-system/carbon#3625]
## Environment
> Operating system
Windows 7

> Browser
Chrome and Firefox

## Detailed description

> What version of the Carbon Design System are you using?

React Storybook, v10, themes

> What did you expect to happen?

In the "current page" of the breadcrumbs, the only means of understanding that the current page is being displayed is to be able to distinguish between the blue and black text. Normally, my expectation would be that the links in the breadcrumbs (which would be the blue text here) would all have underlines, making them clearly identifiable as actionable links. Since the current page would not be expected to have a link, I would understand that this item without a link in the breadcrumb must represent the current page.

If an underline is not going to be employed on the breadcrumb links, then the black text of the current page must have at least a 3:1 relative luminance with the blue links of the actionable breadcrumbs.

> What WCAG 2.1 checkpoint does the issue violate?
https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html

## Additional information
Note that this model is disrupted, since the current page IS a link. That is a separate issue I will capture later, but I mention it here since it makes the current issue a bit messier.

Another option to overcome this challenge is to put a closing slash on the normal breadcrumb view (e.g., Breadcrumb 1 / Breadcrumb 2 / Breadcrumb 3 /). That way, when the current page is included, it would be distinguished by the fact there is no terminating slash e.g., Breadcrumb 1 / Breadcrumb 2 / Breadcrumb 3 / current page).

**Comment by asudoh** [comment]
@dakahn @carbon-design-system/design Would you think of any high contrast design for https://the-carbon-components.netlify.com/component/breadcrumb--current-page.html? Thanks!

**Comment by dakahn** [comment]
Distinguishing current page in the breadcrumb solely by color is a problem for sure and yeah mirroring what @mbgower is saying -- it's also strange that the current page is _still a link_ but could easily be perceived as uninteractable text.

**Comment by aagonzales** [comment]
The current page shouldn't be a link, that's a bug.

**Comment by aagonzales** [comment]
I think we should also just remove "current page" as an option anyways. The current page title should be featured outside of the breadcrumb anyways. Its repetitive to have it in the breadcrumb as well as on the page. Removing it should resolve these issues.

**Comment by asudoh** [comment]
Just to double-check @aagonzales - Does it mean we should simply remove "with current page" demo as well as its associated styles?

**Comment by asudoh** [comment]
Related: #2156

**Comment by aagonzales** [comment]
Actually, someone here in the office brought up some example of when the current page still might be needed. So we need to keep it but not have it as part of the default breadcrumb. 

I'd be fine with this proposed solution, if it solves the problem.

> Another option to overcome this challenge is to put a closing slash on the normal breadcrumb view (e.g., Breadcrumb 1 / Breadcrumb 2 / Breadcrumb 3 /). That way, when the current page is included, it would be distinguished by the fact there is no terminating slash e.g., Breadcrumb 1 / Breadcrumb 2 / Breadcrumb 3 / current page)

![image](https://user-images.githubusercontent.com/11670886/62954258-ad411f80-bdb4-11e9-9f1e-d62fec4741de.png)

**Comment by shixiedesign** [comment]
I remember there was some restrictions the "current" item has to be a link. I'm wondering if we can customize this one specific link instance to have a hover state that's visually the same as default (no underline and use default cursor)?
